bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/productlist/ProductList.java

```diff
@@ -71,21 +71,15 @@
     @Nullable
     String getTitle();
 
-    int getTotalCount();
```

Comment:
Here again an attempt was made to DRY things out a bit by moving all pagination / search result information and pagination functionality into the search result class(es).